src/main/java/com/mate/fruitshop/service/impl/CsvReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.mate.fruitshop.service.impl;
+
+import com.mate.fruitshop.service.CsvReaderService;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReaderServiceImpl implements CsvReaderService {

```

Comment:
```suggestion
public class ReaderServiceImpl implements ReaderService {
```
we can read not only from csv